modules/ols-creating-the-credentials-secret-using-web-console.adoc

```diff
@@ -39,4 +37,22 @@ stringData:
 [NOTE]
 ====
 The YAML element is always `apitoken` regardless of what the LLM provider calls the access details.
-====
\ No newline at end of file
+====
++
+.Example credential secret for {azure-official} {openai}
++
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: v1
+data:
+  client_id: <base64 encoded client id>
+  client_secret: <base64 encoded client secret>
+  tenant_id: <base64 encoded tenant id>
```

Comment:
same comment as above re missing underscores for variables

Reply:
done